modules/private-shared-lib/src/domain/individual_transfer_entity.ts

```diff
@@ -53,18 +54,32 @@ export enum IndividualTransferInternalState {
     TRANSFERS_SUCCESS = 'TRANSFERS_SUCCESS',
 }
 
+export interface IndividualQuoteResponse extends SDKSchemeAdapter.V2_0_0.Outbound.Types.individualQuoteResult {
+    expiration: SDKSchemeAdapter.V2_0_0.Outbound.Types.DateTime;
+}
+
+export interface IndividualTransferResponse extends SDKSchemeAdapter.V2_0_0.Outbound.Types.individualTransferResult {
+    completedTimestamp?: SDKSchemeAdapter.V2_0_0.Outbound.Types.DateTime;
```

Comment:
This is also missing some fields, i.e. `ilpFulfilment`, etc?

Reply:
similar to the above comment:
https://github.com/mojaloop/api-snippets/blob/f4dfd101d68c6a55e71867d167dc765b16129214/docs/fspiop-rest-v1.1-openapi3-snippets.yaml#L3622

and
https://github.com/mojaloop/api-snippets/blob/master/sdk-scheme-adapter/v2_0_0/components/schemas/individualTransferResult.yaml